nfcore_custom.config

```diff
@@ -34,6 +34,7 @@ profiles {
   uzh          { includeConfig "${params.custom_config_base}/conf/uzh.config" }
   prince       { includeConfig "${params.custom_config_base}/conf/prince.config" }
   bigpurple    { includeConfig "${params.custom_config_base}/conf/bigpurple.config" }
+  pasteur      { includeConfig "${params.custom_config_base}/conf/pasteur.config" }
```

Comment:
Would be great if you can do the same here too. It seems to have lost its alphabeticity somehow!

Reply:
I guess we weren't really enforcing it at the moment, so thats why (or I wasn't enforcing it 😆  when reviewing but agree that it helps with general readability...)